apps/sequencer/src/feeds/feeds_slots_manager.rs

```diff
@@ -270,7 +270,7 @@ pub async fn register_feed_with_config(
         }
 
         let new_feed_metadata = FeedMetaData::new(
-            &new_name,
```

Comment:
This is the only place where a reference is taken, so perhaps just avoid the clone a few lines earlier?